lib/biran/configurinator.rb

```diff
@@ -122,6 +122,12 @@ def sanitize_config_files files_list
       end
     end
 
+    def yaml_load data_content
+      # Ruby 3.1 with Psych 4 allows yaml-aliases only in direct manner
+      return YAML.safe_load(data_content, [], [], true) if defined?(Psych::VERSION) && Gem::Version.new(Psych::VERSION) < Gem::Version.new('4.0')
```

Comment:
I went this route so that it would be easy to remove this line once we no longer want to support ruby less than 3.1 basically (or psych less than 4).

Tested in my local against both ruby 2.6.5 and ruby 3.1.2